sdk/nodejs/runtime/closure/createClosure.ts

```diff
@@ -150,6 +150,11 @@ interface Context {
     // __awaiter per .js file that uses 'async/await'.  Instead of needing to generate serialized
     // functions for each of those, we can just serialize out the function once.
     simpleFunctions: FunctionInfo[];
+
+    /**
+     * The resource to log any errors we encounter against.
+     */
+    logResource?: resource.Resource;
```

Comment:
When creating the serialized code for a Lambda/FunctionApp it would be good to associate with a resource to more specifically narrow down where the problem is.